### PR TITLE
Update netatmo.markdown

### PR DESCRIPTION
**Pull request by edwinm**
Restart Home Assistant after changing configuration.yaml

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @edwinm, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20598).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by cgtobi — COMMENTED** [review]
I don't think this is specific to the Netatmo integration.

**Comment by edwinm** [comment]
You're probably right, but I think it's good to be part of the instructions.

**Review by frenck — DISMISSED** [review]
I agree, it doesn't harm; lets just add it 👍

File: source/_integrations/netatmo.markdown [hunk 1]
```diff
@@ -233,7 +233,7 @@ client_secret:
   type: string
 {% endconfiguration %}
 
-**After the client_id and client_secret is added to your `configuration.yaml` you must enable Netatmo through the integrations page.**
+**After the client_id and client_secret is added to your `configuration.yaml` you must restart Home Assistant en then enable Netatmo through the integrations page.**
```
Comment (frenck) [inline source/_integrations/netatmo.markdown]:
```suggestion
**After the `client_id` and `client_secret` are added to your `configuration.yaml` you must restart Home Assistant and then enable Netatmo through the integrations page.**
```

**Review by frenck — APPROVED** [review]

**Pull request merged**